crates/pop-cli/src/commands/test/contract.rs

```diff
@@ -14,6 +14,12 @@ pub(crate) struct TestContractCommand {
 	path: Option<PathBuf>,
 	#[arg(short = 'f', long = "features", help = "Features for the contract project")]
 	features: Option<String>,
+	#[arg(
+		short = 'n',
+		long = "contracts-node",
+		help = "Path to the contracts node to run the e2e tests [default: none]"
+	)]
+	contracts_node_path: Option<PathBuf>,
 }
 
 impl TestContractCommand {
```

Comment:
May I ask why we are taking a reference to `self` in `execute(&self)`? It requires to do a lot of clones while `self` is not used upstream. Not 100% sure but perhaps worth looking into.

Reply:
Fixed in this PR: https://github.com/r0gue-io/pop-cli/pull/217